modules/analyzing-all-violations.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * operating/respond-to-violations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="analyzing-all-violations_{context}"]
+= Analyzing all violations
+
+By viewing the *Violations* page, you can analyze all violations and take corrective action.
+
+.Procedure
+
+. In the {product-title-short} portal, click *Violations*.
+. Choose the appropriate method to view the violations from the drop-down list, which is in the upper left of the page:
+** To display the findings for application workloads, select *Applications view*
+** To display the findings for platform components in OpenShift, select *Platform view*.
+** To display the findings for application workloads and platform components simultaneously, select *Full view*.
+. Optional: Choose the appropriate method to re-organize the information in the *Violations* page:
+** To sort the violations in ascending or descending order, select a column heading.
+** To filter violations, use the filter bar.
+** To see more details about the violation, select a violation in the *Violations* page.
+. Optional: To exclude a policy from the violations list, perform the following steps:
```

Comment:
I believe this only excludes a deployment from within the policy. Charmik, can you confirm?